src/binary/ibc-setup/commands/init.ts

```diff
@@ -0,0 +1,57 @@
+import fs from 'fs';
+import path from 'path';
+
+import { Command } from 'commander';
+import yaml from 'js-yaml';
+
+import { GlobalOptions } from '../types';
+import { resolveOption } from '../utils/resolve-option';
+
+export type Options = GlobalOptions & {
+  src: string;
+  dest: string;
+};
+
+type ProgramOptions = Partial<Options>;
+
+const REGISTRY_FILE = 'registry.yaml';
+
+export function init(_: unknown, program: Command) {
+  const programOptions: ProgramOptions = program.opts();
+
+  function getDefaultHome() {
```

Comment:
This could go top-level, right? Or even in a `utils.ts` file.

Reply:
For sure, however, there's only one usage now and I'd like to keep things as close as possible. I'll lift it up when there'll be a need. Is that fine?

Reply:
Sure. Let's get this working. There will be a few rounds of refactoring as you add some more commands